packages/types/src/identify.ts

```diff
@@ -20,7 +20,7 @@ export enum IdentifyOperation {
   CLEAR_ALL = '$clearAll',
 }
 
-export type ValidPropertyType = number | string | Array<string | number>;
+export type ValidPropertyType = number | string | Array<string | number> | { [key: string]: any };
```

Comment:
can this type be recursigvely tightened? e.g. { key: string: number | string | Array<string | number> | NestedObj }